scripts/build-default.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# This file is part of vs-fltk
+# License: https://github.com/KaruroChori/vs-fltk/blob/master/LICENCE.md
+
+set -e
+
+if [ !-f "src/app/main.cpp" ]; then
```

Comment:
```suggestion
if [ !-f "src/app/main.cpp" ]; then
```
Sorry, just to check how it works